code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -226,6 +226,11 @@ datum/reagent/medicine/adminordrazine/on_mob_life(var/mob/living/carbon/M as mob
 	..()
 	return
 
+datum/reagent/medicine/adminordrazine/nanites
```

Comment:
I think its fine for a admin event.

It can easily be balanced if its found to be too effective once the ERT has been used.